dotnet/samples/KernelSyntaxExamples/Example54_AzureChatCompletionWithData.cs

```diff
@@ -8,19 +8,25 @@
 
 /**
  * This example shows how to use Azure OpenAI Chat Completion with data.
- * More information: <see href="https://learn.microsoft.com/en-us/azure/ai-services/openai/use-your-data-quickstart"/>
  */
 public static class Example54_AzureChatCompletionWithData
 {
+    /* Set-up
```

Comment:
Suggest updating all the comments to use the `/// <summary>` approach.

Reply:
Updated to XML function comments